use-cases/using-SPDK-vhostuser-and-kata.md

```diff
@@ -0,0 +1,153 @@
+# Setup to run SPDK vhost-user devices with Kata Containers and Docker*
+
+- [SPDK vhost-user target overview](#spdk-vhost-user-target-overview)
+- [Install and setup SPDK vhost-user target](#install-and-setup-spdk-vhost-user-target)
+  - [Get source code and build SPDK](#get-source-code-and-build-spdk)
+  - [Run SPDK vhost-user target](#run-spdk-vhost-user-target)
+- [Host setup for vhost-user devices](#host-setup-for-vhost-user-devices)
+- [Launch a Kata container with SPDK vhost-user block device](#launch-a-kata-container-with-spdk-vhost-user-block-device)
+
+## SPDK vhost-user Target Overview
+
+The Storage Performance Development Kit (SPDK) provides a set of tools and
+libraries for writing high performance, scalable, user-mode storage applications.
+
+While, virtio is a efficient way to transport data for virtual environments and
+guests. It is most commonly used in QEMU VMs, where the QEMU itself exposes a
+virtual PCI device and the guest OS communicates with it using a specific Virtio
```

Comment:
Could you change "Virtio" to either "virtio" or "VirtIO" consistently throughout this doc to match the rules in the spell checker?